config/locales/en.yml

```diff
@@ -6,3 +6,4 @@ en:
       invalid_ingested_date: 'must be in the format dd/mm/yyyy'
       invalid_optional_ingested_date: 'must be in the format dd/mm/yyyy or left blank'
       invalid_lot_number: 'is not included in the supplier framework agreement'
+      invalid_dependent_field: '%{value} is not a valid %{attr} for the given %{parent_field_name} of %{parent_field_value}'
```

Comment:
We get values with spaces in, which gives us error messages like 
`Health and Healthcare is not a valid Primary Specialism for the given Service Type of Core`. 

Should we quote these to read something like

`"Health and Healthcare" is not a valid Primary Specialism for the given Service Type of "Core"`?

Should we quote anything at all? Everything? Just values?